### DISCUSSION TITLE
Manual

### DISCUSSION THREAD

**Discussion by MPH90-M** [collin80/SavvyCAN#736] [General]
Hi @all

The complete documentation site is offline, I use SavvyCAN for my Bachelor Thesis and would need the manual.
Is it possible to get it somehow?

Thanks in advance.
Marc

**Comment by collin80** [comment]
The complete help documentation is included with SavvyCAN itself and also can be found on github here: https://github.com/collin80/SavvyCAN/tree/master/help
